dist/proton-player.cjs.js

```diff
@@ -16,6 +16,9 @@ class ProtonPlayerError extends Error {
 }
 
 function debug(...args) {
+  {
+    console.log(`%c[ProtonPlayer]`, 'color: #e26014; font-weight: bold;', ...args);
+  }
```

Comment:
I believe you need to compile the JS bundles with the `production` flag so that these are removed